qir/qat/Apps/Qat/ProfileConfiguration.hpp

```diff
@@ -0,0 +1,292 @@
+        "validation-configuration", ValidationPassConfiguration::fromProfileName("default"));
+
+    // The default profile does not alter the default settings for transformation
+}
+
+inline void configureProvider4bf9Profile(ConfigurationManager& configuration_manager)
+{
+
+    // Setting profile validation configuration
+    configuration_manager.addConfig<ValidationPassConfiguration>(
+        "validation-configuration", ValidationPassConfiguration::fromProfileName("provider_4bf9"));
+
+    // Transformation
+    configuration_manager.disableSectionById("grouping");
+
+    configuration_manager.updateParameter("apply", true);
+    configuration_manager.updateParameter("validate", true);
+    configuration_manager.updateParameter("S", false);
+    configuration_manager.updateParameter("O0", false);
+    configuration_manager.updateParameter("O1", false);
+    configuration_manager.updateParameter("O2", false);
+    configuration_manager.updateParameter("O3", false);
+    configuration_manager.updateParameter("verify-module", true);
+    configuration_manager.updateParameter("experimental", false);
+    configuration_manager.updateParameter("dump-config", true);
+    configuration_manager.updateParameter("add-ir-debug", false);
+    configuration_manager.updateParameter("strip-existing-dbg", false);
+
+    configuration_manager.updateParameter("optimize-result-one", true);
+    configuration_manager.updateParameter("optimize-result-zero", true);
+    configuration_manager.updateParameter("optimize-result-comparison", true);
+    configuration_manager.updateParameter("use-static-qubit-array-allocation", true);
+    configuration_manager.updateParameter("use-static-qubit-allocation", true);
+    configuration_manager.updateParameter("use-static-result-allocation", true);
+    configuration_manager.updateParameter("disable-reference-counting", true);
+    configuration_manager.updateParameter("disable-alias-counting", true);
+    configuration_manager.updateParameter("disable-string-support", true);
+    configuration_manager.updateParameter("disable-record-output-support", false);
+
+    configuration_manager.updateParameter("unroll-loops", false);
+    configuration_manager.updateParameter("allow-partial", true);
+    configuration_manager.updateParameter("allow-peeling", true);
+    configuration_manager.updateParameter("allow-runtime", true);
+    configuration_manager.updateParameter("allow-upper-bound", true);
+    configuration_manager.updateParameter("allow-profile-based-peeling", true);
+    configuration_manager.updateParameter<uint64_t>("full-unroll-count", 1024);
+    configuration_manager.updateParameter<int32_t>("unroll-opt-level", 3);
+    configuration_manager.updateParameter("only-when-forced", false);
+    configuration_manager.updateParameter("forget-scev", false);
+    configuration_manager.updateParameter("always-inline", true);
+    configuration_manager.updateParameter<int32_t>("inlining-parameter", 2147483647);
```

Comment:
How can the reader/user/maintainer make sure that the "_magic number_" `2147483647` is correct? 
May be a hexadecimal representation can make more sense?

Reply:
Fixed.